src/System.Management.Automation/engine/Modules/ModuleUtils.cs

```diff
@@ -233,7 +233,7 @@ internal static IEnumerable<string> GetDefaultAvailableModuleFiles(string topDir
                 foreach (string ext in ModuleIntrinsics.PSModuleExtensions)
                 {
                     string moduleFile = Path.Combine(directoryToCheck, proposedModuleName) + ext;
-                    if (!Utils.NativeFileExists(moduleFile))
+                    if (!File.Exists(moduleFile))
```

Comment:
Same here.
Actually, I see `Utils.NativeFileExists` and `File.Exists` mixed in this file (possibly in that files too). Do we really care about the exception on failure in those cases?

Reply:
Utils.FileExists behavior is the same (no exception is thrown). So it is better to use the standard method.

Yes, we should review all these cases but there are dozens of them to do in this PR or in a new?

Reply:
Yeah. let's do it in a separate PR. For this PR, just do the simple rename replacement: `Utils.NativeFileExists -> Utils.FileExists`.